packages/loader/container-loader/src/connectionManager.ts

```diff
@@ -585,6 +585,14 @@ export class ConnectionManager implements IConnectionManager {
 					this.logger.sendTelemetryEvent({ eventName: "ReceivedClosedConnection" });
 					connection = undefined;
 				}
+				this.logger.sendTelemetryEvent(
+					{
+						eventName: "ConnectedToDeltaStream",
```

Comment:
nit: should we rename this event to "ConnectionReceived"

Reply:
Deal!